Tests/UIKitTests/UIViewExtensionsTests.swift

```diff
@@ -206,6 +206,38 @@ final class UIViewExtensionsTests: XCTestCase {
 		view.addSubviews([UIView(), UIView()])
 		XCTAssertEqual(view.subviews.count, 2)
 	}
+    
+    func testAddSubViewsWithTranslatesAutoresizingMaskIntoConstraints() {
```

Comment:
Can you please use multiple values in the array just to make it a bit clearer that it works on multiple sizes. Also test for an empty array.